app/src/engineering/java/com/hedvig/app/feature/embark/EmbarkStoryTesterActivity.kt

```diff
@@ -0,0 +1,259 @@
+                            .fillMaxWidth(),
+                    ) {
+                        items(viewState.availableMarkets) { market ->
+                            MarketItem(market) {
+                                model.onMarketClick(market)
+                            }
+                        }
+                    }
+                    Text(text = "Stories")
+                    LazyColumn(
+                        verticalArrangement = Arrangement.spacedBy(4.dp),
+                        modifier = Modifier
+                            .padding(horizontal = 16.dp)
+                            .fillMaxWidth(),
+                    ) {
+                        items(viewState.bundles) { bundle ->
+                            EmbarkStoryItem(bundle) {
+                                model.onStorySelected(bundle.storyName)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun EmbarkStoryItem(bundle: BundlesResult.Success.Bundle, onClick: () -> Unit) {
+        Surface(
+            shape = MaterialTheme.shapes.medium,
+            modifier = Modifier.clickable { onClick() }
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .padding(12.dp)
+                    .fillMaxWidth(),
+            ) {
+                Column {
+                    Text(
+                        text = bundle.storyName,
+                        style = MaterialTheme.typography.subtitle1,
+                        modifier = Modifier
+                            .padding(bottom = 8.dp)
+                            .width(300.dp),
+                    )
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun MarketItem(market: Market, onClick: () -> Unit) {
+        Surface(
+            shape = MaterialTheme.shapes.medium,
+            modifier = Modifier.clickable { onClick() }
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .padding(12.dp),
+            ) {
+                Column {
+                    Text(
+                        text = market.name,
+                        style = MaterialTheme.typography.body1,
+                        modifier = Modifier
+                            .padding(bottom = 8.dp),
+                    )
+                }
+            }
+        }
+    }
+}
+
+val embarkStoryTesterModule = module {
+    viewModel { EmbarkStoryTesterViewModel(get(), get(), get()) }
+}
+
+class EmbarkStoryTesterViewModel(
+    private val getBundlesUseCase: GetBundlesUseCase,
+    private val marketManager: MarketManager,
+    private val authenticationTokenService: AuthenticationTokenService,
+) : ViewModel() {
+
+    data class ViewState(
+        val loading: Boolean = true,
```

Comment:
I think this is unused right? Can just remove it since it's just dev tooling if we don't need it in this screen